app/models/image.rb

```diff
@@ -1,17 +1,17 @@
 class Image < ActiveRecord::Base
 
   has_attached_file :content,
-    styles: {
-      medium: "300x300>",
-      thumb: "100x100#",
-      medium_square: "700x500#",
-      facebook: "1200x630>",
-      large: "1920x"
+    :styles => {
+        :medium=> "300x300>",
+        :thumb=> "100x100#",
+        :medium_square=> "700x500#",
+        :facebook=> "1200x630>",
+        :large=> "1920x"
     },
-    convert_options: {
-      all: '-strip -interlace Plane'
+    :convert_options=> {
+        :all=> '-strip -interlace Plane'
     },
-    default_url: "/images/:style/missing.png"
+    :default_url=> "/images/:style/missing.png"
```

Comment:
This was to see whether changing to vintage syntax might help with the issue where Paperclip wasn't dealing properly with the url parameter that had to be specified as a string. All of the tutorials I found where it worked used old syntax. I just left it like that after having tried it out - it made no difference, I was pretty desperate :)